### PR TITLE
CVE-2010-3413 and CVE-2014-7924

### PR DESCRIPTION
**Pull request by curtissimo41**
Initial Pull Request

### CODE REVIEW

File: cves/CVE-2011-1443.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE: 
+CWE: 1021
```
Comment (mesternefeld) [inline cves/CVE-2011-1443.yml]: Please revert this file if it was not worked on fully.

Comment (curtissimo41) [reply cves/CVE-2011-1443.yml]: Changes made.

File: cves/CVE-2011-1443.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE: 
+CWE:
```
Comment (mesternefeld) [inline cves/CVE-2011-1443.yml]: Please fill in a CWE

File: cves/CVE-2011-1443.yml (hunk 1 above)
Comment (mesternefeld) [reply cves/CVE-2011-1443.yml]: I still see it, unfortunately. Maybe it was not reverted properly?

File: cves/CVE-2014-7924.yml [hunk 3]
```diff
@@ -77,9 +82,11 @@ unit_tested:
 
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  answer: 
-  code: 
-  fix: 
+  answer: |
+    The code was run through some tests, and a fix for the problem was
+    implemented after they found where the issue was in the code.
```
Comment (mesternefeld) [inline cves/CVE-2014-7924.yml]: Interesting, so were new tests added?

File: cves/CVE-2014-7924.yml [hunk 4]
```diff
@@ -32,7 +32,12 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: |
+  The vulnerability occured when more than one cursor tries to access the same
+  BLOB in the IndexedDB, resulting in a heap-use-after-free error.
```
Comment (dariuscox) [inline cves/CVE-2014-7924.yml]: Maybe describe what a BLOB is and what heap-use-after-free is.

File: cves/CVE-2014-7924.yml [hunk 5]
```diff
@@ -94,21 +101,25 @@ discovered:
...
 subsystem:
   question: |
     What subsystems was the mistake in?
 
     Look at the path of the source code files code that were fixed to get
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged. Examples: "clipboard", "gpu", "ssl", "speech", "renderer"
-  answer: 
-  name: 
+  answer: Based on the blog post
```
Comment (dariuscox) [inline cves/CVE-2014-7924.yml]: I think this is supposed to be the subsystem

File: cves/CVE-2014-7924.yml [hunk 6]
```diff
@@ -198,4 +213,11 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer: 
+  answer: |
+    The biggest coding mistake that was made with this vulnerability was that
+    the code never checked for duplicate pointers, or whether locations on the
+    heap or in the IndexedDB were freed of their allocated memory or not.
+
+    The fix that was made works properly by limiting access to the heap to only
```
Comment (dariuscox) [inline cves/CVE-2014-7924.yml]:
```suggestion
    The fix that was implemented by limiting access to the heap to only
```

File: cves/CVE-2010-3413.yml [hunk 7]
```diff
@@ -93,21 +104,26 @@ discovered:
...
     What subsystems was the mistake in?
 
     Look at the path of the source code files code that were fixed to get
     directory names. Look at comments in the code. Look at the bug reports how
-    the bug report was tagged. Examples: "clipboard", "gpu", "ssl", "speech", "renderer"
-  answer: 
-  name: 
+    the bug report was tagged. Examples: "clipboard", "gpu", "ssl", "speech",
+    "renderer"
+  answer: Provided in the comments of the bug report.
```
Comment (dariuscox) [inline cves/CVE-2010-3413.yml]: Should be the subsystem

File: cves/CVE-2010-3413.yml [hunk 8]
```diff
@@ -32,38 +32,46 @@ description_instructions: |
...
+repo:
 fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
 - :commit: 706640aae5285ff0843cd038777616ca85525cce
-  :note: ''
+  :note:
 vccs:
 - :commit:
```
Comment (dariuscox) [inline cves/CVE-2010-3413.yml]: No vcc commit, you state that the vcc was the creation of the module, what commit was that?

**Pull request merged**